src/components/WalletTransactionDetails.vue

```diff
@@ -115,11 +115,17 @@ export default class WalletTransactionDetails extends Mixins(TranslationMixin, N
   }
 
   get isComplete(): boolean {
-    return [TransactionStatus.Finalized, 'done'].includes(this.selectedTransaction.status as TransactionStatus);
+    return [TransactionStatus.InBlock, TransactionStatus.Finalized, 'done'].includes(
```

Comment:
`done`?? I forgot, why did we put it here

Reply:
I didn't found this status in available transaction statuses (checked it in codec), so i removed it from here